inception-project/inception
# #822 - KB resources show as URI not as label in statements

**What's in the PR**
* Bugfix for #822 
* send proper selection event depending on the type of the resource when link is clicked

**How to test manually**
* check description in #822, see if bug still occurs
* click on links that point to properties or instances, see if ```Infopanel``` and ```ListPanel``` are displayed correctly.

**Automatic testing**

**Documentation**
